docs/vision-usage.rst

```diff
@@ -214,7 +244,7 @@ in an image.
 
 
 Image Properties
-~~~~~~~~~~~~~~~~
+----------------
```

Comment:
Yeah just to make it match up. But looking at that, I will redo it with the proper markup starting with `H1`'s